OWNERS

```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- wongma7
-- jsafrane
+- kubernetes-csi-approvers
+
 reviewers:
-- wongma7
-- jsafrane
+- kubernetes-csi-reviewers
+
```

Comment:
Please add back Please add wongma7.

Reply:
wongma7 is not in the kubernetes-csi-reviewers group?

Reply:
He is not.